xbmc/Application.cpp

```diff
@@ -2797,6 +2797,7 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
     if(item.GetProperty("StartPercent").isString())
       fallback = atof(item.GetProperty("StartPercent").asString().c_str());
     options.startpercent = item.GetProperty("StartPercent").asDouble(fallback);
+    item.m_lStartOffset = 0;
```

Comment:
I'm confused by this can you explain the logic? Did you test the "happy path", meaning actually having a resume point where you want to resume?

Shouldn't we only set this to 0 if options.startpercent is greather then 0.0f?

Reply:
I'm confused by your confusion. I thought this is what you were talking about :) That's where the property is looked at. I changed it so that m_lStartOffset is not STARTOFFSET_RESUME (-1). 

Then Kodi uses the StartPercent here:
https://github.com/xbmc/xbmc/blob/0978b1dfa3923726841a04b11202eff44552d644/xbmc/cores/VideoPlayer/VideoPlayer.cpp#L1300-L1312

Reply:
> Shouldn't we only set this to 0 if options.startpercent is greather then 0.0f?

I would assume it should always be set because we never want Kodi to load a resume point if StartPercent is set, no?

Reply:
Ah, yeah makes sense. I did not remember those were separate concepts...

Reply:
Ok good. Well this can be merged then. Thanks for your time @Razzeee